pkg/interface/launch/src/js/components/welcome.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+
+export class Welcome extends Component {
+  constructor() {
+    super();
+    this.state = {
+      show: true
+    }
+  }
+
+  render() {
+    let firstTime = window.startupMessage;
+    return (firstTime && this.state.show)
+    ? (
+      <div className={"fl ma2 bg-white bg-gray0-d white-d overflow-hidden " +
+      "ba b--black b--gray1-d pa2 w-100 mw6 lh-copy"}>
+      <p className="f9">You've just come into possession of a virtual computer you own completely. You can run it on a laptop, you can run it on a server, but it doesn't really matter.</p>
+      <p className="f9 pt2">With the key you used to access this space, you can never lose this computer. You can never lose what's on this computer.</p>
+      <p className="f9 pt2">To begin, you should probably pop into a chat and verify there are signs of life in this new place. If you were invited by a friend, you probably already have access to a few.</p>
```

Comment:
I know copy isn't final yet, but in case this doesn't change:

> probably ... probably

```suggestion
      <p className="f9 pt2">To begin, you should pop into a chat and verify there are signs of life in this new place. If you were invited by a friend, you likely already have access to a few.</p>
```

Reply:
Wow, you iced both of them. @Fang- vendetta against the word "probably" lol